test/acceptance/tools/cygnus.py

```diff
@@ -66,6 +68,8 @@
 CKAN_SINK        = u'ckan'
```

Comment:
because, the sink names are constant values, for me is better define in constants, but it is only my opinion. I know that this behavior involves jumps in the code, I plan to do refactoring in the future